tools/packaging/postinstall.sh

```diff
@@ -11,12 +11,15 @@ fi
 cleanInstall() {
     printf "\033[32m Post Install of a clean install\033[0m\n"
 
+    if ! getent group pyroscope >/dev/null 2>&1; then
+        groupadd -r pyroscope
+    fi
     # Create the user
     if ! id pyroscope > /dev/null 2>&1 ; then
-        adduser --system --home /var/lib/pyroscope --shell /bin/false "pyroscope"
+        adduser --create-home --system --home /var/lib/pyroscope -g pyroscope --shell /bin/false "pyroscope"
```

Comment:
I think debian doesn't like `create-home` and only shares `--gid` with the redhat variant:

```suggestion
        adduser --system --home /var/lib/pyroscope --gid $(getent group | grep pyroscope: | awk -F ":" '{ print $3 }') --shell /bin/false "pyroscope"
```

Reply:
I've made a little improvement to your suggestion.
WDYT?